test/utils.unit.js

```diff
@@ -0,0 +1,52 @@
+
+var utils = require('../lib/utils');
+var assert = require('assert');
+
+var isArray = utils.isArray;
+var isObject = utils.isObject;
+
+describe('utils', function() {
+    describe('isArray()', function() {
+        it('---> ok', function() {
+            var result1 = isArray([]);
+            assert.ok(result1);
+        });
+
+        it('---> ok (withouth isArray)', function() {
+            var original = Array.isArray;
+            Array.isArray = undefined;
+            var result1 = isArray([]);
+            assert.ok(result1);
+            Array.isArray = original;
+        });
+
+        it('---> ko', function() {
```

Comment:
Can you please give these tests titles that describe a bit more what they're doing?  At first I thought `ko` was a typo.

Reply:
No problem I will change them